CodeGeneration/Sources/SyntaxSupport/KeywordSpec.swift

```diff
@@ -59,6 +60,26 @@ public struct KeywordSpec {
   ) {
     self.name = name
     self.experimentalFeature = experimentalFeature
+    self.experimentalFeature2 = nil
+    self.isLexerClassified = isLexerClassified
+  }
+
+  /// Initializes a new `KeywordSpec` instance.
+  ///
+  /// - Parameters:
+  ///   - name: A name of the keyword.
+  ///   - experimentalFeature: The experimental feature the keyword is part of, or `nil` if this isn't for an experimental feature.
+  ///   - or: A second experimental feature the keyword is also part of, or `nil` if this isn't for an experimental feature.
+  ///   - isLexerClassified: Indicates if the token kind is switched from being an identifier to a keyword in the lexer.
+  init(
+    _ name: String,
+    experimentalFeature: ExperimentalFeature,
+    or experimentalFeature2: ExperimentalFeature,
```

Comment:
And then make the initializer take a variadic parameter: `experimentalFeature: ExperimentalFeature…`,